src/Utilities/Compiler/Options/EditorConfigOptionNames.cs

```diff
@@ -219,11 +219,11 @@ internal static partial class EditorConfigOptionNames
         public const string LinqChainMethods = "linq_chain_methods";
 
         /// <summary>
-        /// Boolean option to configure the assumption that IEnumerable type arguments would be enumerated by method invocation or not.
+        /// Boolean option to configure the assumption that IEnumerable type parameters would be enumerated by method invocation or not.
         /// It does not affect linq_chain_methods.
         /// Configurable rule: CA1851 (https://docs.microsoft.com/visualstudio/code-quality/ca1851).
         /// </summary>
-        public const string AssumeMethodEnumeratesArguments = "assume_method_enumerates_arguments";
+        public const string AssumeMethodEnumeratesParameters = "assume_method_enumerates_parameters";
```

Comment:
I rename the word 'argument' to 'parameters' because I feel it's more precise.
For example:
```
void Bar(IEnumerable<int> input)
{
   OtherMethod(input);
}

void OtherMethod(object o) {}
```
If using the term 'argument', it feels like 'OtherMethod' would enumerated the `input`.
But in fact we only want to flag if `OtherMethod` has a signature like
`void OtherMethod(IEnumerable<T> i) {}`